sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,933 @@
+{
+    "title": "Appeals - Student Dependants",
+    "name": "studentDependantsAppealPartTime",
+    "path": "studentdependantsappealparttime",
+    "type": "form",
+    "display": "form",
+    "tags": [
+        "common"
+    ],
+    "components": [
+        {
+            "key": "dependantsInformationTitle",
+            "label": "Dependants Information",
+            "input": false,
+            "tag": "h3",
+            "attrs": [
+                {
+                    "value": "",
+                    "attr": ""
+                }
+            ],
+            "className": "category-header-medium primary-color",
+            "content": "Dependants Information",
+            "type": "htmlelement",
+            "hideLabel": true,
+            "tags": [],
+            "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+            },
+            "properties": {},
+            "lockKey": true
+        },
+        {
+            "clearOnHide": false,
+            "key": "dependantsAppealPanel",
+            "input": false,
+            "title": "Dependants Information Part Time Applications Panel",
```

Comment:
Minor. can you remove the part time from the title? - I know is hidden but could be confusing for a dev looking in future.

Reply:
Also on the file name and the path.